decidim-admin/lib/decidim/admin/test/admin_members_shared_examples.rb

```diff
@@ -1,26 +1,26 @@
 # frozen_string_literal: true
 
-shared_examples "manage participatory process members examples" do
+shared_examples "manage admin members examples" do
   let(:other_user) { create(:user, organization:, email: "my_email@example.org") }
 
-  let!(:member) { create(:member, user:, privatable_to: participatory_process) }
+  let!(:member) { create(:member, user:, participatory_space: participatory_space) }
```

Comment:
I wonder why this wasn't caught by rubocop. 
```suggestion
  let!(:member) { create(:member, user:, participatory_space:) }
```

Reply:
Nice catch! It seems like it is a rule that we don't have enabled

What we have is EnforcedShorthandSyntax: either (the default when there isn't any added).

So 

```ruby

# good
{foo: foo, bar: bar}

# good
{foo: foo, bar:}

# good
{foo:, bar:}

```

I'll open a PR adding the rule and correcting all of these offenses

Reply:
Opened a new PR on https://github.com/decidim/decidim/pull/15822 for fixing that